_includes/markdown/Systems.md

```diff
@@ -201,54 +201,71 @@ The three commonly encountered MySQL replication types are Master-Slave, Master-
 
 Most Linux servers have a few tmpfs mounts already. The `/dev/shm` mount is a tmpfs mount for efficiently storing temporary files for programs and we can set MySQL to use this with the tmpdir variable in the ```my.cnf``` file. tmpfs only uses space in RAM when files exist (it isn’t preallocated), but if the WordPress database is very busy, beware that MySQL could use as much RAM as is allocated for ```/dev/shm``` (which is normally 50% of total RAM). In most WordPress workloads, MySQL only uses a few megabytes in temporary tables on disk, but if ```/dev/shm``` usage were to grow, it could quickly cause an out of memory situation. Take this into account when tuning MySQL for memory usage. ```/dev/shm``` size should be monitored if used for the MySQL tmpdir.
+
+<h2 id="https" class="anchor-heading">HTTPS{% include Util/top %}</h2>
+
+New sites should implement HTTPS (Hypertext Transfer Protocol Secure) unless there is a specific reason not to, even if they don't handle sensitive communications. HTTPS provides many benefits over HTTP such as:
+
+* **Security:** HTTPS creates an encrypted channel between your browser and a website's server. Without encryption, anything transferred between the website and browser could be intercepted by a 3rd party. This is an issue for usernames, passwords, payment information and anything else sent over a form.
+
+* **Performance:** The HTTP/2 web protocol includes numerous performance benefits such as allowing multiple requests to be sent over a single connection, which only works on websites running on HTTPS.
+
+* **SEO:** While Google's search algorithms are ever-evolving, [Google is moving towards making sure that HTTPS sites will get a page rank boost](https://webmasters.googleblog.com/2014/08/https-as-ranking-signal.html). Using HTTPS automatically gives sites ranking improvements over non-HTTPS websites.
+
+* **Browser Compatibility:** Modern browsers such as Google Chrome and Firefox display a 'Not Secure' message on any page where data is entered into a form that isn't protected by HTTPS. This is a poor experience for users that could result in an increased bounce rate.
+
+#### **HTTPS Best Practices**
+* **Mixed Content** - Mixed content occurs when HTML is loaded over a secure HTTPS connection and requests resources such as images and scripts to be loaded over an insecure HTTP connection. Most browsers display warnings about this type of content and prevents this content from loading on a page. Before implementing a new website on HTTPS, it's important to run a mixed content scanning tool that will crawl and scan an HTTPS-enabled website for mixed content. WordPress plugins should also be utilized that will check for insecure content from within the WordPress editor interface.
```

Comment:
If we have tools we recommend for mixed content scanning, and plugins we like for this, it would be nice to link to them or mention them here.  It would make this section much more useful.

Reply:
I linked to one tool that has been used on other projects in the past, but I think we could do a better job at finding WordPress plugins that do this. I could have sworn we had one that we managed but it looks like that project was archived.